### PR TITLE
Add browser comparison pages

### PR DESCRIPTION
**Pull request by craigcook**
Five new pages comparing Firefox to other major browsers, as well as a shared lang file.

We'll be handling French and German in-house. This is a LOT of text so all other locales are lowest priority, no deadline, and purely opt-in so localizers should only work on these pages if they want to.

Lang files added in https://github.com/mozilla-l10n/www.mozilla.org/pull/371

### CODE REVIEW

**Comment by craigcook** [comment]
D'oh, looks like I can't add the shared.lang to the no_active_tag list until it actually exists. Should I remove it and add it later, or we can just rerun the Travis build after the other PR is merged?

**Comment by flodolo** [comment]
> D'oh, looks like I can't add the shared.lang to the no_active_tag list until it actually exists. Should I remove it and add it later, or we can just rerun the Travis build after the other PR is merged?

You added a file to the no_active_tag, but you didn't define it in the list of supported files for mozilla.org

**Comment by craigcook** [comment]
🤦‍♂

File: app/config/sources.inc.php [hunk 1]
```diff
@@ -133,6 +134,48 @@
         ],
         'supported_locales' => $mozillaorg,
     ],
+    'firefox/compare/shared.lang' => [
```
Comment (peiying2) [inline app/config/sources.inc.php]: Nit pick: this goes after the new Safari entry.

**Comment by craigcook** [comment]
Reordered and squashed

**Pull request merged**